tests/unit/tasks/models/tasksModel.spec.ts

```diff
@@ -1,10 +1,32 @@
 import jsLogger from '@map-colonies/js-logger';
+import nock from 'nock';
+import { TaskHandler as QueueClient } from '@map-colonies/mc-priority-queue';
 import { TasksManager } from '../../../../src/tasks/models/tasksManager';
-
-let tasksManager: TasksManager;
+import { configMock, registerDefaultConfig, clear as clearConfig } from '../../../mocks/configMock';
+import { IJobManagerConfig } from '../../../../src/common/interfaces';
 
 describe('TasksManager', () => {
+  let jobManagerClient: TasksManager;
```

Comment:
taskManager instead of 'jobManagerClient'

Reply:
changed it